frontend/src/Component/OrderCard/OrderCardContent.tsx

```diff
@@ -0,0 +1,49 @@
+import {ServiceOrder} from "../../model/ServiceOrder";
+import {Box, CardContent, Typography} from "@mui/material";
+import ComponentTable from "../ComponentTable/ComponentTable";
+import React from "react";
+
+type OrderCardContentProps = {
+    order: ServiceOrder
+}
+
+export default function OrderCardContent(props: OrderCardContentProps){
```

Comment:
or directly (if this works for you 😄) :
`export default function OrderCardContent(props: ServiceOrder)`

Reply:
This would theoretically work, but the problem is that I then have to pass all the 7 properties of ServiceOrder to the component, which is very tedious. This way I can just pass the Order as a whole.